### ISSUE TITLE
Баг в api AppMarket`а

### ISSUE THREAD

**Issue by ke46138** [IgorTimofeev/MineOS#653]
**Describe the bug**
При достижении лимита опубликованных приложений в AppMarket (как я понял в 8 штук) при опубликовании 9 приложения вылазит ошибка о том что нельзя больше опубликовать приложение, но если на другом аккаунте попытаться опубликовать приложение, то оно не опубликуется и вылезет ошибка о том что приложение уже существует в AppMarket, но его там нет. Можно проверить попытавшись опубликовать приложение с названием Black EFI.
В общем появляются фантомные приложения.

**To Reproduce**
1. Опубликовать 8 приложений на аккаунте
2. Нажать кнопку опубликовать ПО в AppMarket
3. Заполнить всё как нравится (например название Black EFI)
4. Увидеть ошибку о том что больше публиковать приложения нельзя
5. Выйти из аккаунта
6. Зайти на второй аккаунт
7. Нажать кнопку опубликовать ПО в AppMarket
8. Заполнить всё как нравиться, но оставить название с прошлого раза (например Black EFI)
9. Увидеть ошибку о том что приложение уже существует

**Expected behavior**
Опубликование приложения с прошлым названием

**Screenshots**
Лень делать, если надо сделаю

**Minecraft and OpenComputers version**
Minecraft 1.12.2
OpenComputers v1.8.6

**Additional context**
Скорее всего проблема в API

**Comment by CoolCat467** [comment]
Yes, this would indeed be an issue on app market's server side code.

**Comment by Smok1e** [comment]
Это не баг. Публикация с названием "Black EFI" уже была опубликована другим пользователем, но не в разделе приложений, а в скриптах

![Image](https://github.com/user-attachments/assets/ae9389e4-9940-402b-8736-60f08ef19ab7)
